.github/workflows/autoupdate.yml

```diff
@@ -3,9 +3,12 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    types: [ labeled ]    
 jobs:
   autoupdate:
     name: autoupdate
+    if: ${{ github.event.label.name == 'autoupdate' }}
```

Comment:
It is redundant to have both `PR_LABELS` and the `if` condition that checks for the label name being "autoupdate".